### DISCUSSION TITLE
Sourcing from multisite Wordpress installation

### DISCUSSION THREAD

**Discussion by jondujaka** [gatsbyjs/gatsby#31038] [Help]
Hi,

I am trying to source from a multisite installation, but I can't figure out if that is possible.
I've been searching around, and I found this https://github.com/gatsbyjs/gatsby/issues/12608, but for me it doesn't allow two instances of the `gatsby-source-wordpress` plugin.

Does anyone have any pointers on how to set up sourcing for a situation like this?

**Comment by sarahannnicholson** [comment]
I am also wondering the same thing. With Gatsby source wordpress on v5 now I thought maybe this issue [redux issue](https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/issues/58) 
got resolved. But cannot seem to find any documentation for a multisite config.

**Comment by CobyPear** [comment]
I am also wondering if this feature fell off the roadmap?

I think to workaround this you can fetch data directly from your WordPress backend, but then you don't have the data in Gatsby's GraphQL layer so it's not a great solution depending on what you're after.
